lisp/ess-roxy.el

```diff
@@ -68,7 +68,7 @@
 (require 'outline)
 (eval-when-compile
   (require 'cl-lib))
-(require 'essddr "ess-rd.el")
```

Comment:
This change has already been made in `next`

Reply:
Maybe this one should go on next actually, or after you merge next.